pkg/controller/common/license/trial.go

```diff
@@ -21,6 +21,8 @@ import (
 )
 
 const (
+	AutogenLicenseIssuer = "Elastic k8s operator"
```

Comment:
Maybe `ECKLicenseIssuer`? `autogen` feels like there is some kind of magic I need to think about.